doc/overview/consensus/1tx-reconfig.rst

```diff
@@ -92,3 +92,34 @@ Instead, upon retiring from a network, retired leaders still respond to requests
 For crash fault tolerance, this means the following: Before the reconfiguration the network could suffer f_C0 failures. After the reconfiguration, the network can suffer f_C1 failures. During the reconfiguration, the network can only suffer a maximum of f_C0 failures in the old **and** f_C1 failures in the new configuration as a failure in either configuration is unacceptable. This transitive period where the system relies on both configurations ends once the new configuration's leader's global commit index surpasses the commit that included the reconfiguration as described above.
 
 In our example above, the election timeout on Node 1 simply expires and causes Node 1 to call for an election, which it wins immediately.
+
+Retirement details
+~~~~~~~~~~~~~~~~~~
+
+Retirement of a node runs through four phases, as indicated by the following diagram. It starts with a reconfiguration transaction (RTX) and it involves 
+two additional elements of state:
+
+- Retirement index (RI): Index at which node is set to ``Retired`` in ``public:ccf.gov.nodes.info``
+- Retirement Committable Index (RCI): Index at which the retirement transaction first becomes committable, ie. the first signature following the transaction.
+
+A node permanently transitions to the ``Completed`` phase once it has observed commit reaching its Retirement Committable Index.
+
+.. mermaid::
+
+    graph TB;
+        Active-- RTX executes -->Started
+
+        subgraph Retired
+            Started-- RTX commits -->Ordered;
+            Ordered[Ordered: RI set]
+            Ordered-- Signature -->Signed;
+            Signed[Signed: RCI set]
+            Signed-- RCI commits -->Completed;            
+            Ordered-.->Started
+            Signed-.->Ordered
+        end
+
+Until the very last phase (``Completed``) is reached, a retiring leader will not actively attempt to change leadership, but elections may still happen as usual.
```

Comment:
This is a slightly odd sentence, because a leader never attempts to change leadership in Raft (or in our CFT). I think "will continue to act as leader" is better, although there is a slight caveat: a retiring leader having observed RCI on itself will not execute further transactions, but will continue to send append entries/heartbeats.